src/main/include/Constants.h

```diff
@@ -206,7 +206,7 @@ namespace CONSTANTS
         CONSTANTS::DRIVE::ROBOT_MAX_ANGULAR_SPEED;
     constexpr units::radians_per_second_squared_t TRAJ_MAX_ANGULAR_ACCELERATION{
         std::numbers::pi};
-    static constexpr auto WHEEL_CIRCUMFERENCE = 12.11_in / 1.0_tr;
+    static constexpr auto WHEEL_CIRCUMFERENCE = 8.277511961722488_in / 1.0_tr;
```

Comment:
This is a bug. Change this back to 12.11 and change `SensorToMechanismRatio` in `SwerveModule.cpp` to 6.75.
`SensorToMechanismRatio` still has the old L4 value of 4.722.